CHANGELOG.rst

```diff
@@ -6,7 +6,27 @@ This document describes changes between each past release.
 1.4.2 (unreleased)
```

Comment:
Yep :) also, PyFxA have been merged so I'll remove the dependency link here but let the list and everything so it's easier to add later.